Assets/MixedRealityToolkit.Tests/EditModeTests/SpatialAwareness/SpatialAwarenessSystemTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using Microsoft.MixedReality.Toolkit.Tests.Services;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests.SpatialAwarenessSystem
+{
+    public class SpatialAwarenessSystemTests
+    {
+        private const string TestSpatialAwarenessSysteProfilePath = "Assets/MixedRealityToolkit.Tests/EditModeTests/Services/TestProfiles/TestMixedRealitySpatialAwarenessSystemProfile.asset";
+
+        [TearDown]
+        public void TearDown()
+        {
+            TestUtilities.ShutdownMixedRealityToolkit();
+        }
+
+        [Test]
+        public void TestGetSpatialAwarenessSystem()
+        {
+            TestUtilities.InitializeMixedRealityToolkitAndCreateScenes(true);
+
+            // Retrieve Input System
```

Comment:
>Input System [](start = 24, length = 12)

spatial awareness system

Reply:
I think there are a few more copy pastas down below as well

---
In reply to: [314129827](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5659#discussion_r314129827) [](ancestors = 314129827)